assets/js/header/css/external_header.css

```diff
@@ -0,0 +1,152 @@
+.the-header {
```

Comment:
try to centralize the external css locations ... move them out out django and into webpack